paper/report_final.tex

```diff
@@ -0,0 +1,578 @@
+
+Since the performance of the test statistic of linear models are largely depend on the normality assumption on the independent variables, the check of normality assumption is indispensable. We choose the following methods for normality assumption:
+
+\begin{itemize}
+\item{QQ plot} The quantile-quantile plot (QQ plot) is the most commonly used visualization method to check the validity of a distribution assumption. The basic idea is to compute the empirical quantile the and compare it with the theoretically expected value of a normal distribution. If the data follow a normal distribution, then the points on the Q-Q plot would fall on a straight line. 
+\item{Residuals vs. fits plot} A residuals vs. fits plot is another most frequently created plot. Under the normality assumption, the residuals should be independent and scatted around.
+\end{itemize}
+
+\subsubsection{ANOVA test}
+
+In our dataet, each participant repeated the test for three times, in other words there are data of 3 runs for each subject. Before collapsing the three runs into one model, we need to check the assumption whether they are indiscriminate.  To do this, we perform an ANOVA test on each subject to check the difference of means across runs. The significance of an AVOVA test may show that there are difference across runs, thus we may seek other methods to eliminate the influence due to the variance across runs.
+
+\subsubsection{Multiple Test correction}
+
+In statistical inference for fMRI data, usually we have more than tens of thousands of hypothesis tests, thus massive multiple correction problems. Using thresholds without correction could be problematic. Common multiple correction methods (such as Bonferroni) require adjusting the p-values. However, imposing high statistical thresholds that may mask voxels that do have real effects. To avoid this loss, we use uncorrected threshold and choose the threshold on a case-by-case basis.
+
+
+\subsubsection{Cluster analysis}
+
+We use K-means clustering to detect the homogeneity of regions. We concatenate the betas obtained from linear models and the coordinate of voxels as the clustering vector (dimension = 16 + 3 = 19). Then do cluster analysis for beta gains and beta loss separately to detect regions activated by increasing beta losses and beta gains.
+
+
+\section{Results}
+
+\subsection{Behavioral analysis}
+
+We performed statistical analysis using both Python and R (The original paper use R package to fit the Logistic models). We use the library \emph{scikit-learn} in Python and the \emph{glm} function in \emph{stats} in R to fit the models. Models from two library yields the same results. We shows the plot of the behavioral loss aversion $\lambda$ for every subject (median=1.94, mean=2.18, min=0.99, max=0.75). This result is consistent with that of the paper, which indicate that participants are indifferent to gambles whose gain are approximately twice as the loss. 
+
```

Comment:
I think it doesn't matter to mention this. We do use R to do the exploratory analysis.